editoast/src/views/v2/path/pathfinding.rs

```diff
@@ -392,19 +329,13 @@ impl From<TrackOffsetExtractionError> for PathfindingResult {
     }
 }
 
-/// extract locations from path items
-pub async fn extract_location_from_path_items(
-    conn: &mut DbConnection,
-    infra_id: i64,
-    path_items: &[PathItemLocation],
-) -> Result<TrackOffsetResult> {
-    let (trigrams, ops_uic, ops_id) = collect_path_item_ids(path_items);
-
-    let (uic_to_ops, trigrams_to_ops, ids_to_ops) =
-        retrieve_op_from_locations(conn, infra_id, &ops_uic, &trigrams, &ops_id).await?;
-
+/// Extract locations from path items
+pub fn extract_location_from_path_items(
```

Comment:
This could be a method of `PathItemCache`

Reply:
Done